casper/src/main/rholang/MakePoS.rho

```diff
@@ -0,0 +1,111 @@
+contract @"MakePoS"(@purse, @minimumBond, @maximumBond, @initBonds, return) = {
```

Comment:
So this `return` channel must be public so that anyone can submit a deploy that uses `this` right?

Reply:
It'll be registered with the name registry to ensure users can send messages to it, but not listen on the same name, intercepting its messages.

Reply:
Sounds good